comms/discovery/server/server.go

```diff
@@ -74,7 +75,8 @@ func NewServer(jsc nats.JetStreamContext, proc *rpcz.RPCProcessor) *ChatServer {
 	g.GET("/chats/:id/messages", s.getMessages)
 	g.POST("/mutate", s.mutate)
 
-	g.GET("/chat-permissions", s.getChatPermissions)
+	g.GET("/chats/permissions", s.getChatPermissions)
+	g.GET("/chats/blocked", s.getChatBlocked)
```

Comment:
_super_ nit feel free to ignore but: `blockees`? `blocked-users`? (I think in REST it should be a noun)